internal/handler/auth_handlers.go

```diff
@@ -129,14 +125,14 @@ func LoginPerson(c *fiber.Ctx) error {
 	if email != "" {
 		voucherOwnerPerson, errDB = queries.GetPersonByEmail(ctx, email)
 		if errDB != nil {
-			return fiber.NewError(fiber.StatusBadRequest, "DB person with email "+email+"  not exist error "+errDB.Error())
+			return fiber.NewError(fiber.StatusBadRequest, "DB person with email ["+email+"] not exist error ["+errDB.Error()+"]")
```

Comment:
In Go it is considered good form not to use the "+" sign for string concatenation. Use fmt.Sprintf instead.